.github/workflows/pr-loadtest.yml

```diff
@@ -0,0 +1,65 @@
+name: PR Load Test
+
+# Trigger the test Manually additionally provide PR number.
+on:
+  workflow_dispatch:
+    inputs:
+      pr_number:
+        description: 'Pull Request Number'
+        required: true
+
+env:
+  IMG: europe-docker.pkg.dev/kyma-project/dev/telemetry-manager:PR-${{ github.event.inputs.pr_number }}
+  K8S_VERSION: "1.27"
+
+jobs:
+  load-test:
+    strategy:
+      max-parallel: 3
+      matrix:
+        argument: ["-n pr-lt-traces -t traces", "-n pr-lt-traces-m -t traces -m true", "-n pr-lt-traces-b -t traces -b true", "-n pr-lt-metrics -t metrics", "-n pr-lt-metrics-m -t metrics -m true", "-n pr-lt-metrics-b -t metrics -b true"]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          ref: 'refs/pull/${{ github.event.inputs.pr_number }}/head'
+          repository: ${{ github.repository }}
+
+      - name: Setup Golang
+        uses: "./.github/template/setup-golang"
+
+      # Generate the cluster name to be used for cluster creation. The cluster-name should be `pr-lt-traces`
+      - name: Setup gardener cluster name
+        run:  echo "GARDENER_CLUSTER_NAME=$(echo '${{ matrix.argument }}' | awk '{print $2}')" >> $GITHUB_ENV
+
+      # save gardener kubeconfig to a temp file in order to pass it to the command
+      - name: Save service account to file
+        shell: bash
+        run: 'echo "$GARDENER_SA" > /tmp/gardener-sa.yaml'
+        env:
+          GARDENER_SA: ${{ secrets.GARDENER_SA }}
+
+      # provision cluster
+      - name: Provision Gardener
+        run: make provision-gardener
+        env:
+          GARDENER_SECRET_NAME: ${{ secrets.GARDENER_SECRET_NAME }}
+          GARDENER_PROJECT: ${{ secrets.GARDENER_PROJECT }}
+          GARDENER_SA_PATH: /tmp/gardener-sa.yaml
+          GARDENER_K8S_VERSION: ${{ K8S_VERSION }}
+
+      - name: Deploy Istio Module
+        run: hack/deploy-istio.sh
+
+      - name: Deploy Telemetry
+        run: hack/deploy-telemetry.sh
+
+      # Run the load test
+      - name: Run Load test
+        working-directory: docs/contributor/telemetry-load-test
+        run: ./assets/run-load-test.sh ${{ matrix.argument }}
```

Comment:
Shouldn't we move it to the hack folder as well and adjust the docu, now that we are using it really in the repo